build/SharedFx.targets

```diff
@@ -87,23 +87,23 @@
         <AdditionalProperties>
           MetapackageName=Microsoft.AspNetCore.App;
           MetapackageReferenceType=AppMetapackage;
-          LockToExactVersions=true
+          MetapackageDependencyVersionRangeType=MajorMinor
         </AdditionalProperties>
       </_MetapackageBuilderProject>
 
       <_MetapackageBuilderProject Include="$(MSBuildProjectFullPath)">
         <AdditionalProperties>
           MetapackageName=Microsoft.AspNetCore.All;
           MetapackageReferenceType=AllMetapackage;
-          LockToExactVersions=false
+          MetapackageDependencyVersionRangeType=Minimum
         </AdditionalProperties>
       </_MetapackageBuilderProject>
 
       <_MetapackageBuilderProject Include="$(MSBuildProjectFullPath)">
         <AdditionalProperties>
           MetapackageName=Microsoft.AspNetCore.Analyzers;
           MetapackageReferenceType=Analyzer;
-          LockToExactVersions=true
+          MetapackageDependencyVersionRangeType=Exact
```

Comment:
Now that I think about it, I don't recall why this is locked to exact versions. We should re-check our assumption that this metapackages should remain locked. @pranavkm ?

Reply:
We're not shipping Microsoft.AspNetCore.Analyzers at the moment, right? Is this something we plan to ship at some point in 2.1.x?

Reply:
Spoke to @pranavkm, seems like we probably don't want to lock the versions in this metapackage either. I think we should probably relax this to major.minor as well. This package is experimental for now so I'm less concerned about it but I still don't think locking to exact versions makes sense.

Reply:
Ok, I'll switch this to MajorMinor as well.